styles.css

```diff
@@ -205,3 +210,84 @@ figure:hover figcaption {
   width: 10px;
   height: 17px;
 }
+
+/* ABOUT US section */
+
+.about-us {
+  margin: 190px 100px 0px 100px;
+}
+
+.about-us h2 {
+  text-align: center;
+  font-size: 36px;
+}
+
+.amazing-taiwan {
+  color: #ee6c4d;
+}
+
+ .about-us .flex-row-container {
+  justify-content: space-around;
+}
+
+.information {
+  margin-top: 108px;
+  line-height: 15px;
```

Comment:
`line-height` figmoje nurodyta kitokia reiškmė. Ir dar klausimas - koks yra font-size šitoje vietoje? Nes jeigu jis didesnis nei 15px, tai nesueina dimersijos ir dėl to darosi sunku išdėlioti elementus taip, kaip norėtųsi.